src/clients/add-on_auto.rb

```diff
@@ -272,6 +272,7 @@ def main
               end
 
               Ops.set(repos, [found_at, "name"], name)
+              Ops.set(repos, [found_at, "priority"], Ops.get_integer(prod, "priority", -1))
```

Comment:
Here I think that could be easier and understandable to use more ruby-like syntax

```ruby
repos[found_at]["priority"] = prod["priority"]
```

But I chose to follow the current code style.

Reply:
As this is for master we do not need to be afraid of some more cleanup. Feel free to change the line and even the lines around.

Reply:
Ok, I'm changing some of them in a separated PR, as I was talking with @kobliha in the IRC channel :)